stock_move_ignore_last_purchase_date/models/product_product.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2023 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ProductProduct(models.Model):
+    _inherit = "product.product"
+
+    @api.depends("stock_move_ids.state", "man_last_purchase_date")
+    def _compute_last_purchase_date(self):
```

Comment:
@AungKoKoLin1997 Please avoid a complete override of methods.  Make the original method extendable by splitting the relevant part into another method.